cnxarchive/sql/search/author.part.sql

```diff
@@ -9,15 +9,15 @@ SELECT
   %({0})s::text||'-::-author' as key
 FROM
   latest_modules AS m,
-  users AS u 
+  users AS u
 WHERE
   u.username = ANY (m.authors)
   AND
-  (u.first_name ~* req(%({0})s::text)
+  (u.first_name ~* regexp_replace(req(%({0})s::text), ' ', '.+.')
    OR
-   u.last_name ~* req(%({0})s::text)
+   u.last_name ~* regexp_replace(req(%({0})s::text), ' ', '.+.')
    OR
-   u.full_name ~* req(%({0})s::text)
+   u.full_name ~* regexp_replace(req(%({0})s::text), ' ', '.+.')
```

Comment:
I think we should either have `.+` (my preference) or `.*.`, not `.+.`, otherwise it wouldn't match "jill miller".  And why remove the global flag?